searchguard/rolesmapping.py

```diff
@@ -152,3 +152,22 @@ def modify_rolemapping(role, properties, action="replace"):
 
     # No merge or split action, overwrite existing properties:
     _send_api_request(role, properties)
+
+
+def list_rolemappings_for_user(user, roles):
+    """Get list of rolemappings that contain the given user.
+
+    :param str user: Name of user
+    :param list roles: List of rolemappings to be checked for the given user
+    :returns list: list of rolemappings with the given user
+    :raises: ViewRoleMappingException
+    """
+    user_rolemappings = []
+
+    for role in roles:
```

Comment:
Can we make roles (rolemappings) optional?  Would be nice if the behaviour without rolesmappings as argument performs a full search on all rolemappings.

Reply:
Sounds like a useful feature, will add this.